tests/requirements.in

```diff
@@ -1,7 +1,11 @@
+ansible<=2.6
+pytest<=3.6
```

Comment:
Please do something like `pytest >= 3.6.4 && > 3.7.2` to have
- a lower bound
- the ability to update requirements easier later

Also, consider adding a comment linking to https://github.com/pytest-dev/pytest-bdd/issues/250

Reply:
Something like "pytest >= 3.6.4 && < 3.7  # pytest-dev/pytest-bdd#25" is ok ? (not sure the issue will be solved for 3.7.2)

Reply:
In my comment that was the actual link (which I'd prefer), but Github shortened it :)

Indeed, we can't know it's fixed in `> 3.7.2`, but at the same time we don't necessarily want to restrict us to `< 3.7`. All we know is it doesn't work with `3.7 <= X <= 3.7.2`. If `3.7.3` fixes the issue, and we update requirements, then CI will pass. Or it will fail, so the bounds would need to be adapted again.